TestCases/compliance-level-3/0004-lending/0004-lending.dmn

```diff
@@ -616,7 +616,7 @@
                     </inputExpression>
                 </input>
                 <input id="_1d794320-6e6c-46bf-b86d-de073822d6f6" label="Pre-bureauAffordability">
-                    <inputExpression typeRef="string">
+                    <inputExpression typeRef="boolean">
```

Comment:
With BKM parameters defined as:
```
            <formalParameter typeRef="tRiskCategory" name="Pre-bureauRiskCategory"/>
            <formalParameter typeRef="boolean" name="Pre-bureauAffordability"/>	            
            <formalParameter typeRef="number" name="Age"/>	            
```
I agree this second table input is a `boolean`. Technically also previous parameter ("Pre-bureauRiskCategory") could be amended to reflect from `string` to `tRiskCategory`, wdyt?